### PR TITLE
Show payment success messages on the loading screen

### PR DESCRIPTION
**Pull request by torotil**

### CODE REVIEW

File: campaignion_foundation.css [hunk 1]
```diff
@@ -97,3 +97,11 @@ form.share-page .form-item-message-format {
 form.share-page textarea#edit-message-value {
   min-height: 18em;
 }
+
+// Hide payment success messages and pull them into the ajax loading dummy.
```
Comment (katzmo) [inline campaignion_foundation.css]: This file is pure CSS, comments need to be formatted as `/* … */` :wink: 

As for the comment itself, the styles only hide the message on the form, the JS puts them into the dummy…

File: campaignion_foundation.css [hunk 2]
```diff
@@ -97,3 +97,11 @@ form.share-page .form-item-message-format {
...
+form .payment-success {
+  display: none;
+}
+.payment-success {
+  text-align: center;
+}
```
Comment (katzmo) [inline campaignion_foundation.css:104]: To be on the safe side (e.g. in the unlikely case that webform-ajax-slide is not enabled), I’d suggest to target `.webform-ajax-slide .payment-success` and `.webform-ajax-slide-loading-dummy .payment-success`.

File: js/jquery.webform-ajax-slide.js [hunk 3]
```diff
@@ -70,6 +70,12 @@
       // Set dummy dimensions to current container dimensions.
       $loadingdummy.css({height: $slide.height() + 'px', width: $slide.width() + 'px'});
 
+      // Fetch payment status messages if they exist.
+      $loadingdummy.find('.payment-success').remove();
+      ajax.form.find('.payment-success').each(function () {
+        $('<p class="payment-success">').html($(this).html()).appendTo($loadingdummy);
+      });
```
Comment (katzmo) [inline js/jquery.webform-ajax-slide.js:77]: Although `<p>` is the semantically correct markup, it does not work best with our default styles :grimacing: If we want to go the lazy way, perhaps we should use `<div class="payment-success form-label">` instead? :thinking:

**Review by katzmo — CHANGES_REQUESTED** [review]

File: campaignion_foundation.css (hunk 1 above)
Comment (torotil) [reply campaignion_foundation.css]: Seems I forgot what the proper comment syntax is for CSS. I’ve changed that and rephrased it also.

File: campaignion_foundation.css (hunk 2 above)
Comment (torotil) [reply campaignion_foundation.css:104]: I’ve changed the commit accordingly.

File: js/jquery.webform-ajax-slide.js (hunk 3 above)
Comment (torotil) [reply js/jquery.webform-ajax-slide.js:77]: I’m happy to follow your suggestion. I’d just use whatever works quickest in this case.

File: campaignion_foundation.css (hunk 1 above)
Comment (katzmo) [reply campaignion_foundation.css]: Thank you!

**Review by katzmo — APPROVED** [review]
